io/disk/Avago_storage_adapter/avago9361_vd.py.data/avago9361_vd.yaml

```diff
@@ -1,8 +1,9 @@
 cenario:
     contorller: 0
```

Comment:
`self.controller = int(self.params.get('controller', default='0'))` but in yaml it is %s/"contorller"/controller